toolchains.xml

```diff
@@ -39,14 +46,4 @@
       <jdkHome>${JDK8_HOME}</jdkHome>
     </configuration>
   </toolchain>
-  <toolchain>
-    <type>jdk</type>
-    <provides>
-      <version>1.9</version>
```

Comment:
Sorry, that was not intended. This is the first time that I have used git and I made a mistake.